src/Model/EventModel.php

```diff
@@ -40,10 +42,11 @@ public function __construct(array $config ) {
         );
         $this->properties = $config['properties'];
         // Can pass in unix timestamp if prefixed with '@'. Else just let it parse the date
-        $this->time = !empty($config['time']) ?
-            new DateTime(
-                is_int($config['time']) ? '@' . $config['time'] : $config['time']
-            ) : null;
+        $time = (array) new DateTime(
```

Comment:
DateTime parses a ton of different date formats but it's definitely possible someone will pass in something invalid. We should probably catch any exceptions and throw a klaviyo exception